src/components/dashboard/SendQRSummary.js

```diff
@@ -70,8 +80,13 @@ const SendQRSummary = (props: AmountProps) => {
         <Section.Row style={styles.sectionRow}>
           <Section.Title style={styles.headline}>Summary</Section.Title>
           <View style={styles.sectionTo}>
-            <Avatar size={90} />
+            <Avatar
+              size={90}
+              style={{ backgroundColor: 'white', borderRadius: '50%' }}
```

Comment:
this default style should be handled by Avatar component if needed

Reply:
Done